src/index.html

```diff
@@ -60,6 +60,7 @@ <h1 data-i18n="covid-19-tracker">Japan COVID-19 Coronavirus Tracker</h1>
         <div class="lang-picker">
           <span class="lang-picker-title" data-i18n="languages">Languages:</span>
```

Comment:
Since it will now be a dropdown, it should be changed to `Language`. I'm not terribly worried about changing it from plural to singular in every language right now.

Reply:
Fixed